src/js/modules/main/components/Navbar.js

```diff
@@ -46,10 +44,15 @@ const Navbar = ({ classes,
   <Toolbar style={styles.toolbar}>
     <ToolbarGroup key={1}>
       <IconButton onClick={openNavDrawer}>
-        <NavigationMenu color={blue50} />
+        <NavigationMenu color={"White"} />
       </IconButton>
-      <Link to="/" className={classes.toolbarTitle}>
-        <ReactSVG className={classes.masthead} path="/img/masthead.svg" />
+      <Link
+        to="/"
+        className={classes.mastheadLink}
+      >
+        <div className={classes.masthead}>
+          The Spectator
+        </div>
       </Link>
```

Comment:
can you call these `brandingLink` and `branding`? (1) That's how I do it in CFA and (2) Masthead is what I call the condensed-nav component in CFA.